### ISSUE TITLE
Combobox with allowCustom and filterable not firing valueChange when backspaced

### ISSUE THREAD

**Issue by draunkin** [telerik/kendo-angular#849]
I'm submitting a bug report for the kendo angular combobox.

currently when you have the below configuration (as a modified version of your current plunker)
it fires the valueChange event when you add text or modify it. But when you backspace out the text and leave the control it does not fire.

```
import { Component } from '@angular/core';

@Component({
  selector: 'my-app',
  template: `
    <kendo-combobox 
		[(value)]="selected"
		[data]="listItems" 
		[allowCustom]="allowCustom"
		[filterable]="true"
		(valueChange)="valueChanged($event)"
		(filterChange)="onFiltered($event)">
    </kendo-combobox>
  `
})
export class AppComponent {
    public selected: string = '';
    public allowCustom: boolean = true;
    public listItems: Array<string> = ["Baseball", "Basketball", "Cricket", "Field Hockey", "Football", "Table Tennis", "Tennis", "Volleyball"];
    valueChanged($event){
      console.log($event);
    }
    onFiltered(filter){
         //do stuff here
     }
}
```

when you change this to remove [filterable] it works as expected and when you backspace out the text the valueChange event fires correctly

This happens on the current version:  "kendo-angular-dropdowns": "1.1.3",

My browser is Chrome on windows, 64bit,  59.0.3071.115 

I know it is not anything in my build process as I can reproduce it easily with the plunker example when I choose open as plunker from the combobox demo page

**Comment by ipeshev** [comment]
Fixed in v1.2.2-dev.201709281629

**Comment by draunkin** [comment]
Great thanks I'll check it out
